demo/api/src/config/config.ts

```diff
@@ -72,6 +72,13 @@ export function createConfig(processEnv: NodeJS.ProcessEnv) {
             },
             storageDirectoryPrefix: envVars.BLOB_STORAGE_DIRECTORY_PREFIX,
         },
+        sentry:
+            envVars.SENTRY_DSN && envVars.SENTRY_ENVIRONMENT
```

Comment:
IMO we should have a prop SENTRY_ENABLED as this would allow easier toggling of the whole feature

Reply:
disabling sentry if there is no DSN defined is imho perfectly fine. Why add yet another env var?

Reply:
We had this discussion several times 😅 

In case you quickly want to disable Sentry (e.g production problems), I think its better to toggle a boolean instead of removing the DSN. In case you want to re-enable it, you have to look up the value for the DSN.

But I'm fine with it... so we can leave it as it is.

Reply:
For the Cdn Guard we also settled on a single variable, so we should do the same here 😁